services/common/src/redux/createAppSlice.ts

```diff
@@ -4,3 +4,8 @@ import { buildCreateSlice, asyncThunkCreator } from "@reduxjs/toolkit";
 export const createAppSlice = buildCreateSlice({
   creators: { asyncThunk: asyncThunkCreator },
 });
+
+export const rejectHandler = (action) => {
+  console.log(action.error);
+  console.log(action.error.stack);
```

Comment:
Intentional to leave the console logs here?

Reply:
This one yeah. We should probably have a _better_ handler for errors, but it's what we're currently doing in slices and I thought it better to at least make it a _shared_ function.